README.md

```diff
@@ -201,7 +203,51 @@ meta.add_annotation(100, 200, metadata = {
 })
 
 # check for mistakes & write to disk
-meta.tofile('example.sigmf-meta') # extension is optional
+meta.tofile('example_cf32.sigmf-meta') # extension is optional
+```
+
+Now lets add another SigMF Recording and associate them with a SigMF Collection:
+
+```python
+from sigmf import SigMFCollection
+
+data_ci16 = np.zeros(1024, dtype=np.complex64)
+
+#rescale and save as a complex int16 file:
+data_ci16 *= pow(2, 15)
+data_ci16.view(np.float32).astype(np.int16).tofile('example_ci16.sigmf-data')
+
+# create the metadata for the second file
+meta_ci16 = SigMFFile(
+    data_file='example_ci16.sigmf-data', # extension is optional
+    global_info = {
+        SigMFFile.DATATYPE_KEY: 'ci16_le', # get_data_type_str() is only valid for numpy types
+        SigMFFile.SAMPLE_RATE_KEY: 48000,
+        SigMFFile.DESCRIPTION_KEY: 'All zero complex int16 file.',
+        SigMFFile.VERSION_KEY: sigmf.__version__,
+    }
+)
+meta_ci16.add_capture(0, metadata=meta.get_capture_info(0))
+meta_ci16.tofile('example_ci16.sigmf-meta')
+
+collection = SigMFCollection(['example_cf32.sigmf-meta', 'example_ci16.sigmf-meta'],
+        metadata = {'collection': {
+            SigMFCollection.AUTHOR_KEY: 'sigmf@sigmf.org',
+            SigMFCollection.DESCRIPTION_KEY: 'Collection of two all zero files.',
+            SigMFCollection.VERSION_KEY: sigmf.__version__,
+        }
+    }
+)
+collection.tofile('example_zeros.sigmf-collection')
+```
+
+The SigMF Collection and its associated Recordings can now be loaded like this:
+
+```python
+from sigmf import sigmffile
+collection = sigmffile.fromfile('example_zeros')
```

Comment:
Is there a way to get a list of stream_names in the collection, if so this would be the perfect place to show how, since a lot of folks might know they are opening a collection but not know the names of each stream

Reply:
Done, with a simple list comprehension to get all the sigmf files also.